Makefile

```diff
@@ -33,3 +33,12 @@ clean:
 	rm -f .coverage
 	find . -name \*.pyc -delete
 .PHONY: clean
+
+api-doc:
+	mkdir api-ref/build api-ref/rst
```

Comment:
this will fail if the folder already exits - use `mkdir -p`

Reply:
Did I ask for your review? :rage: 
JK, thanks :)